google-api-client/src/main/java/com/google/api/client/googleapis/auth/oauth2/DefaultCredentialProvider.java

```diff
@@ -256,7 +258,6 @@ private GoogleCredential getCredentialUsingWellKnownFile(
     }
   }
 
-
   private boolean runningOnAppEngine() {
     if (getEnvEquals("GAE_ENV", "standard")) {
```

Comment:
Strike these 3 lines

Reply:
Sorry, could you double check the line? This comment shows as referring to an empty line delete. Screenshot below:

<img width="675" alt="screen shot 2018-04-27 at 8 25 07 am" src="https://user-images.githubusercontent.com/3584893/39370730-84ac5448-49f4-11e8-9421-8abc0cf69acd.png">